src/views/IDUploader/index.jsx

```diff
@@ -92,7 +92,7 @@ const IDUploader = () => {
             titleTypographyProps={{ align: 'center' }}
             className={styles.ID_card_header}
           />
-          <CardContent>
+          <CardContent sx={{ padding: 5 }}>
```

Comment:
Probably best practice to put `css` adjustments within a .css file. However, this is a very isolated change so I think this is more of a preference for @EjPlatzer.

Reply:
I think it's a matter of opinion which way is better in this case. I'm fine with merging this as is.